### PR TITLE
doc/website/blog: announcing v0.11.0

### PR DESCRIPTION
**Pull request by rotemtam**

### CODE REVIEW

File: doc/website/blog/2023-05-08-atlas-v011-kubernetes-operator.md [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+### What are Kubernetes Operators?
+
+Kubernetes has taken the cloud infrastructure world by storm mostly thanks to its declarative API. When working
+with Kubernetes, developers provide their cluster's desired configuration to the Kubernetes API, and Kubernetes
+is responsible for reconciling the actual state of the cluster with the desired state. This allows developers to
+focus on the desired state of their cluster, and let Kubernetes handle the complexities of how to get there.
+
+This works out incredibly well for stateless components, such as containers, network configuration and access
+policies. The benefit of stateless components is that they can be replaced at any time, and Kubernetes can 
+simply create a new instance of the component with the desired configuration. For stateful resources, such as databases
```
Comment (hilakashai) [inline doc/website/blog/2023-05-08-atlas-v011-kubernetes-operator.md:36]:
```suggestion
simply create a new instance of the component with the desired configuration. For stateful resources, such as databases, 
```

File: doc/website/blog/2023-05-08-atlas-v011-kubernetes-operator.md [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+![](https://user-images.githubusercontent.com/1522681/236139615-1d10feea-8b19-46a2-905b-b614883c48c8.png)
+
+The Atlas Kubernetes Operator supports [declarative migrations](https://atlasgo.io/concepts/declarative-vs-versioned#declarative-migrations).
+In declarative migrations, the desired state of the database is defined by the user and the operator is responsible
+for reconciling the desired state with the actual state of the database (planning and executing `CREATE`, `ALTER`
+and `DROP` statements).
+
+### Diffing policies
+
+One of the common objections to applying declarative workflows to databases is that it is that there are often
```
Comment (hilakashai) [inline doc/website/blog/2023-05-08-atlas-v011-kubernetes-operator.md:62]:
```suggestion
One of the common objections to applying declarative workflows to databases is that there are often
```

File: doc/website/blog/2023-05-08-atlas-v011-kubernetes-operator.md [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
+multiple ways to achieve the same desired state. For example, if you are running a Postgres database, you may want
+to add an index to a table. Depending on your circumstances, you may want to add this index with our without the `CONCURRENTLY`
+option. When using a declarative workflow, you supply _where_ you want to go, but not _how_ to get there. 
+
+To address this concern, we have introduced the concept of "diff policies" to Atlas. Diff policies allow you
+to customize the way Atlas plans database schema changes for you. For example, you can define a diff policy that
+will always add the `CONCURRENTLY` option to `CREATE INDEX` statements. You can also define a diff policy that
+a policy that will skip certain kinds of changes (for example `DROP COLUMN`) altogether.
```
Comment (hilakashai) [inline doc/website/blog/2023-05-08-atlas-v011-kubernetes-operator.md:70]:
```suggestion
will skip certain kinds of changes (for example `DROP COLUMN`) altogether.
```

File: doc/website/blog/2023-05-08-atlas-v011-kubernetes-operator.md [hunk 4]
```diff
@@ -0,0 +1,308 @@
...
+Diff policies are especially valuable when using the Atlas Kubernetes Operator, as they allow you to customize
+and constrain the way the operator manages your database to account for your specific needs.  We will see an example
+of this below.
+
+### Demo time!
+
+Let's see the Atlas Kubernetes Operator in action. In this demo, we will use the Atlas Kubernetes Operator to manage
+a MySQL database running in a Kubernetes cluster. 
+
+In this example, we will create a MySQL database and apply a schema to it. After installing the
```
Comment (hilakashai) [inline doc/website/blog/2023-05-08-atlas-v011-kubernetes-operator.md:96]:
```suggestion
After installing the
```
Sounds redundant after the last sentence you wrote.

File: doc/website/blog/2023-05-08-atlas-v011-kubernetes-operator.md [hunk 5]
```diff
@@ -0,0 +1,308 @@
...
+      );
+```
+In the example above we added a `policy` section to our `AtlasSchema` resource. In this section, we defined a `diff`
+policy that will skip `DROP COLUMN` statements. In addition, we dropped the `short_bio` column from our schema.
+In addition, we dropped the `short_bio` column from our schema. Let's apply the updated schema:
+
+```bash
+kubectl apply -f schema.yaml
+```
+Next wait for the operator to reconcile the desired state with the actual state of the database:
```
Comment (hilakashai) [inline doc/website/blog/2023-05-08-atlas-v011-kubernetes-operator.md:205]:
```suggestion
Next, wait for the operator to reconcile the desired state with the actual state of the database:
```

File: doc/website/blog/2023-05-08-atlas-v011-kubernetes-operator.md [hunk 6]
```diff
@@ -0,0 +1,308 @@
...
+
+### What is the Atlas Kubernetes Operator?
+
+The Atlas Kubernetes Operator is a Kubernetes controller that uses Atlas to manage
+your database schema. The Atlas Kubernetes Operator allows you to define the desired schema
+and apply it to your database using the Kubernetes API.
+
+### Declarative schema migrations
+
+![](https://user-images.githubusercontent.com/1522681/236139615-1d10feea-8b19-46a2-905b-b614883c48c8.png)
```
Comment (a8m) [inline doc/website/blog/2023-05-08-atlas-v011-kubernetes-operator.md:53]: Don't forget to upload it before merging.

**Review by hilakashai — COMMENTED** [review]
🚀

**Review by a8m — APPROVED** [review]

File: doc/website/blog/2023-05-08-atlas-v011-kubernetes-operator.md (hunk 6 above)
Comment (rotemtam) [reply doc/website/blog/2023-05-08-atlas-v011-kubernetes-operator.md:53]: It's already there

**Pull request merged**